website/docs/terms/dry.md

```diff
@@ -2,8 +2,10 @@
 id: dry
 title: DRY
 displayText: DRY  
-hoverSnippet: "Don’t Repeat Yourself - a coding ideal to work towards that can be applied at several levels: a variable, definition of business / transformation logic, joining two models, etc."
+hoverSnippet: Don’t Repeat Yourself - a coding ideal to work towards that can be applied at several levels (e.g. a variable, definition of business / transformation logic, joining two models, etc.) 
 ---
 
-Don’t Repeat Yourself - a coding ideal to work towards that can be applied at several levels: a variable, definition of business / transformation logic, joining two models, etc. Benefits include ease of maintenance and understanding.
+Don’t Repeat Yourself - a coding ideal to work towards that can be applied at several levels: a variable, definition of business / transformation logic, joining two models, etc. 
```

Comment:
@runleonarun Thank you for taking the time to review these so thoughtfully! I know that was no small ask. I really appreciate it. I'll reply to your suggestions here:

> A few general suggestions:
> 
> * We should probably come up with a [content type ](https://github.com/dbt-labs/product-docs-team/blob/main/content-types.md#content-types) for glossary terms so they have a consistent structure. (A glossary term content type.)

We totally should! I give this a stab next week and send to you.

> * Building on the idea of a content type for glossary terms, it would be nice if all definitions had a similar layout. Some seem to be missing Examples, Further reading, and tl;dr's.

Addressed this point [in this comment](https://github.com/dbt-labs/docs.getdbt.com/pull/1335#discussion_r856502137), lmk how that sits with you!

> * Some of the longer definitions feel like content you'd see in a guide or a help page.  I know this is part of your vision, but we might want to tighten up the definitions and link out to a guide content, eventually.

Yeah, it's definitely unconventional. If the users find it confusing, we can definitely course correct. That said, would you feel comfortable with the longer ones going out in their current state?

> * I'd also really like to see the tl;dr's moved to the top. I think they'd be more impactful there.

Addressed this [in this comment](). Lmk how you're feeling on that.

> * I noticed some content written in passive voice.  I commented on a few sentences. The active voice is much easier for non-native English speakers to understand.

That's a great point. I'll go through and remove the passive voice as much as possible. 

> * We should probably change the notes and tips to be consistent with other notes instead of
>   > code callout

Not too sure what you're referring to here. Can you give me an example?

> * I see both 1st and second person used. Can we standardize on second person?

Yeah, that's a good callout. These were written with the assumption that there would be an author byline, so the first and second person switching makes sense in that context. Now I realize the docs doesn't have that functionality. That said, would you be open to putting a contributor list at the top or bottom of the page with a link to their LinkedIn/Twitter? I think it's important to credit people for their writing, especially if we want to encourage external contribution. Let me know how that sits with you.